pkg/service/model_build_load_balancer_test.go

```diff
@@ -17,6 +17,8 @@ import (
 	"sigs.k8s.io/aws-load-balancer-controller/pkg/model/elbv2"
 )
 
+const lbAttrsDeletionProtectionEnabled = "deletion_protection.enabled"
```

Comment:
you could group it together with the rest of the constants in model_build_load_balancer.go

Reply:
I could but its used only in test and I don't think we should put constants used only in test in production source code if that makes sense

Reply:
ok